src/pages/index.astro

```diff
@@ -5,14 +5,41 @@ import "../index.scss";
 
 <Layout title="Welcome to Astro.">
   <main>
-<section class="hero container">
-  <div class="hero__content">
-    <h1 class="hero--title">Travel without limits!</h1>
-      <p class="hero--description">
-        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta ut facere aliquam impedit ullam minusreprehenderit quia in laudantium sint.</p>
-       <button class="hero--cta">Check offer</button>
-  </div>
+    <section class="hero container">
+      <div class="hero__content">
+        <h1 class="hero--title">Travel without limits!</h1>
+        <p class="hero--description">
+          Lorem, ipsum dolor sit amet consecte tur adipisicing elit. Soluta ut
+          facere aliquam impedit ullam minusreprehenderit quia in laudantium
+          sint.
+        </p>
+        <button class="hero--cta">Check offer</button>
+      </div>
+      <div id="modal" class="modal-wrapper">
+        <div class="modal-container">
+          <div class="modal-header">
+            <button class="modal-icon--close">
+              <img src="../cross.svg" alt="cross icon" />
+            </button>
```

Comment:
What about screen readers? We should add some explanation dedicated for those devices.

Reply:
We have tailwind on board, you can use a span with `sr-only` class to fix it

Reply:
Added